internal/edittree/edittree.go

```diff
@@ -0,0 +1,1135 @@
+package edittree
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"sort"
+	"strings"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/internal/edittree/bitvector"
+)
+
+// --- Edit Tree ---
```

Comment:
I like this piece of code documentation a lot! One markup-related note, I believe it'll read nicer on pkg.go.dev when using the godoc comment format, e.g. for [headings](https://tip.golang.org/doc/comment#headings)

Reply:
I've rolled in some doc comment updates! :smile:  I've found https://pkg.go.dev/golang.org/x/pkgsite/cmd/pkgsite great for testing locally how the comments will render on pkg.go.dev.